src/auto/agents/manager.rs

```diff
@@ -40,28 +56,47 @@ r#"
 Personality: {personality}
 
 You will be given one task.
-Your goal is to split that task up into smaller tasks.
-Your goal is to delegate those subtasks, one at a time.
+Your goal is to understand the task, split it into smaller tasks when necessary.
+When parsing user instructions, please prioritize the command the user instructed you to use to do the job, when splitting the tasks into subtasks please consider your current capabilities pool: 
+- Browse (commands: browse_url)
+- Google (commands: google_search)
+- File System (commands: file_write, file_append, file_list, file_read)
+- Wolfram (commands: wolfram)
+- ChatGPT (commands: ask_chatgpt, reset_chatgpt)
+- NewsAPI (commands: news_search)
+- Wikipedia (commands: wikipedia_search, wikipedia_browse)
+
+Your goal is to map the subtasks to your capabilities, delegate those subtasks, one at a time.
```

Comment:
tell the agent to stay within capabilities scope
ask the agent to go straight/quickly to commands

Reply:
Adding the capabilities directly has been tried before, but lead to the problem of the manager over specifying what tools to use, and hurting the organization of the Manager and Employee.

In addition, the way you hardcoded it into the prompt subverts the way that toggling plugins on and off works.